### PR TITLE
Create homework2.py

### PR DESCRIPTION
**Pull request by eugenesamozdran**

### CODE REVIEW

File: homework/jenya_s/homework2.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from math import exp, pi, sqrt
+from sys import argv
+
+x, m, s = float(argv[1]), float(argv[2]), float(argv[3])
```
Comment (aodarc) [inline homework/jenya_s/homework2.py]: use `map` function to simplify converting to **float** (also slice:)

File: homework/jenya_s/homework2.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+def f(x):
+    print (exp(-((x-m)**2)/((s**2)*2))/(sqrt(2*pi)*s))
```
Comment (aodarc) [inline homework/jenya_s/homework2.py]: function should return Expression

**Comment by eugenesamozdran** [comment]
Please check the updated file

File: homework/jenya_s/homework2.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+from math import exp, pi, sqrt
+from sys import argv
+
+x, m, s = map(float, (argv[1], argv[2], argv[3]))
```
Comment (aodarc) [inline homework/jenya_s/homework2.py]: `map(float, argv[1:4])`

**Pull request merged**